recipes/obsidiantools/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "obsidiantools" %}
+{% set version = "0.10.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/obsidiantools-{{ version }}.tar.gz
+  sha256: b6e9f4f774885a13c10b1905a5d8092cb29b5704f7da5b34c15aa9b2a2230b96
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+  run:
+    - python >=3.9
+    - markdown
+    - pymdown-extensions
+    - html2text
+    - pandas
+    - numpy
+    - networkx
+    - python-frontmatter
+    - beautifulsoup4
+    - bleach
+    - lxml
+
+test:
+  imports:
+    - obsidiantools
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/mfarragher/obsidiantools
+  summary: Obsidian Tools - a Python interface for Obsidian.md vaults
+  dev_url: https://github.com/mfarragher/obsidiantools
+  license: BSD-3-Clause-Modification
```

Comment:
It seems like upstream is closer to https://spdx.org/licenses/BSD-3-Clause.html Please verify if that is correct.